android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -63,6 +64,11 @@ public void setMetadata(ReadableMap metadataMap) throws JSONException {
         Radar.setMetadata(RNRadarUtils.jsonForMap(metadataMap));
     }
 
+    @ReactMethod
+    public void setLogLevel(int level) {
+      Radar.setLogLevel(Radar.RadarLogLevel.fromInt(level));
```

Comment:
hmm does this `fromInt` mapping fn exist? seems like that's what the build is complaining about?

Reply:
@jsani-radar Yep, it's there in Kotlin, just not exposed in Java. Needs a `@JvmStatic` annotation which we're adding in v3.1. That said, wonder if we should support passing a string instead of an int and use an if statement to determine the correct enum value? @brettguenther did this in our new Flutter plugin and it may be more readable/usable. In other words, support `Radar.setLogLevel('info')` instead of `Radar.setLogLevel(3)`

Reply:
Now that we've shipped the Flutter SDK, let's just support passing a string instead (see Flutter impl: https://github.com/radarlabs/flutter-radar/blob/master/ios/Classes/RadarFlutterPlugin.m#L136). We can do the same across all of the JS libs including Cordova and Capacitor. Bypasses this issue and is a better developer experience anyway I think.